newIDE/app/src/InstancesEditor/FullSizeInstancesEditorWithScrollbars.js

```diff
@@ -107,6 +55,18 @@ const FullSizeInstancesEditorWithScrollbars = (props: Props) => {
 
   const forceUpdate = useForceUpdate();
 
+  const canvasRectangle = React.useMemo(() => new Rectangle(), []);
+
+  if (editorRef.current) {
+    const elementBoundingRect = editorRef.current.getBoundingClientRect();
+    canvasRectangle.set({
+      left: elementBoundingRect.left,
+      top: elementBoundingRect.top,
+      right: elementBoundingRect.right - SCROLLBAR_DETECTION_WIDTH,
```

Comment:
if one comes back from outside the canvas onto the scrollbars, do they appear?

Reply:
Yes, because the event listener is one the whole canvas